buildpack/build_test.go

```diff
@@ -27,1335 +27,1325 @@ import (
-							assertLogEntry(t, logHandler, expected)
+							for _, file := range []string{
+								"build-env-cnb-layers-dir-A-v1",
+								"build-env-cnb-platform-dir-A-v1",
+								"build-env-cnb-bp-plan-path-A-v1",
+							} {
+								contents := h.Rdfile(t, filepath.Join(appDir, file))
+								if contents != "unset" {
+									t.Fatalf("Expected %s to be unset; got %s", file, contents)
+								}
+							}
 						})
-					})
-				})
-
-				when("labels", func() {
-					it("should include labels", func() {
-						h.Mkfile(t,
-							"[[labels]]\n"+
-								`key = "some-key"`+"\n"+
-								`value = "some-value"`+"\n"+
-								"[[labels]]\n"+
-								`key = "some-other-key"`+"\n"+
-								`value = "some-other-value"`+"\n",
-							filepath.Join(appDir, "launch-A-v1.toml"),
-						)
-
-						br, err := bpTOML.Build(buildpack.Plan{}, config, mockEnv)
-						if err != nil {
-							t.Fatalf("Unexpected error:\n%s\n", err)
-						}
-
-						if s := cmp.Diff(br, buildpack.BuildResult{
-							LaunchBOM: nil,
-							Labels: []buildpack.Label{
-								{Key: "some-key", Value: "some-value"},
-								{Key: "some-other-key", Value: "some-other-value"},
-							},
-							MetRequires: nil,
-							Processes:   []launch.Process{},
-							Slices:      []layers.Slice{},
-						}); s != "" {
-							t.Fatalf("Unexpected:\n%s\n", s)
-						}
-					})
-				})
-
-				when("processes", func() {
-					it("should include processes and use the default value that is set", func() {
-						h.Mkfile(t,
-							`[[processes]]`+"\n"+
-								`type = "some-type"`+"\n"+
-								`command = "some-cmd"`+"\n"+
-								`default = true`+"\n"+
-								`[[processes]]`+"\n"+
-								`type = "web"`+"\n"+
-								`command = "other-cmd"`+"\n",
-							// default is false and therefore doesn't appear
-							filepath.Join(appDir, "launch-A-v1.toml"),
-						)
-						br, err := bpTOML.Build(buildpack.Plan{}, config, mockEnv)
-						if err != nil {
-							t.Fatalf("Unexpected error:\n%s\n", err)
-						}
-						if s := cmp.Diff(br, buildpack.BuildResult{
-							LaunchBOM:   nil,
-							Labels:      []buildpack.Label{},
-							MetRequires: nil,
-							Processes: []launch.Process{
-								{Type: "some-type", Command: "some-cmd", BuildpackID: "A", Default: true},
-								{Type: "web", Command: "other-cmd", BuildpackID: "A", Default: false},
-							},
-							Slices: []layers.Slice{},
-						}); s != "" {
-							t.Fatalf("Unexpected metadata:\n%s\n", s)
-						}
-					})
-
-					it("should set the working directory", func() {
-						h.Mkfile(t,
-							"[[processes]]\n"+
-								`working-dir = "/working-directory"`,
-							filepath.Join(appDir, "launch-A-v1.toml"),
-						)
-						br, err := bpTOML.Build(buildpack.Plan{}, config, mockEnv)
-						h.AssertNil(t, err)
-						h.AssertEq(t, len(br.Processes), 1)
-						h.AssertEq(t, br.Processes[0].WorkingDirectory, "/working-directory")
-					})
```

Comment:
Moved to https://github.com/buildpacks/lifecycle/blob/b71e534defe5a2bca5eed1a738f7b8bc52522153/buildpack/build_test.go#L815-L825